xla/service/gpu/ir_emission_utils.h

```diff
@@ -88,6 +88,18 @@ inline constexpr absl::string_view kUncompilableFusion =
 
 inline constexpr absl::string_view kTopKCustomCallTarget = "__gpu$TopK";
 
+inline constexpr absl::string_view
+    kDynamicSliceFusionWithStaticAddressComputationConfigName =
+        "address_computation";
+inline constexpr absl::string_view
+    kDynamicSliceFusionWithDynamicAddressComputationConfigName =
+        "dynamic_address_computation";
+
+std::optional<std::string> GetCustomFusionConfigName(
```

Comment:
Exposed utils should have docstrings, and those should be in the `.h` file (it seems like both functions have docstrings in the `.cc` file for some reason).

Reply:
Thanks, I added them there by mistake.